source/clear-linux/tutorials/web-server-install/web-server-install.rst

```diff
@@ -71,7 +71,74 @@ browser as shown in figure 1.
     Confirmation the Apache service is running.
 
 The :file:`index.html` file is located in the :file:`/usr/share/httpd/htdocs`
-directory of your host system.
+directory of your host system.  We will copy this file into a new location 
+after we modify the configuration.
+
+Change the Default Configuration and Data Directory
+===================================================
+
+|CL| is designed to be a stateless operating system which means that you will
+need to create an optional configuration file.  The default location of the
+Apache configuration file, httpd.conf, is located in the
```

Comment:
Keep going with the use of the :file: role:
:file:`httpd.conf`
And below in lines 83 & 84:
:file:`/etc/share/defaults/httpd`
:file:`.conf`